app/src/org/commcare/dalvik/activities/EntitySelectActivity.java

```diff
@@ -623,7 +626,18 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 createSortMenu();
                 return true;
             case MENU_MAP:
+                try
+                {
+                    Class.forName("com.google.android.maps.MapActivity");
+                }
+                catch (Throwable e1) {

```

Comment:
Isn't it a good idea to be explicit about the exception being caught?
In this case "ClassNotFoundException"